server/database/queries/events/index.js

```diff
@@ -0,0 +1,15 @@
+const getEventsDate = require('./getEventsWithDate');
```

Comment:
It'd be nice to make the naming of these files consistent - in each case think about exactly what the function does and give it a descriptive name, otherwise the code becomes very difficult when you have a lot of generic sounding function names.

I'd suggest in this module the function names could be of the format `performGetAllEventsQuery`, or something like that